app/models/header_lookup.rb

```diff
@@ -1,25 +1,21 @@
 class HeaderLookup
-  BASE_SPARQL_MESH_URI ="https://id.nlm.nih.gov/mesh/sparql?format=JSON&limit=10&inference=true&query=PREFIX%20rdfs"\
-    "%3A%20%3Chttp%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%3E%0D%0APREFIX%20meshv%3A%20%3Chttp%3A%2F%2Fid.nlm."\
-    "nih.gov%2Fmesh%2Fvocab%23%3E%0D%0APREFIX%20mesh2018%3A%20%3Chttp%3A%2F%2Fid.nlm.nih.gov%2Fmesh%3E%0D%0A%0D%0ASELECT"\
-    "%20%3Fd%20%3FdName%0D%0AFROM%20%3Chttp%3A%2F%2Fid.nlm.nih.gov%2Fmesh%3E%0D%0AWHERE%20%7B%0D%0A%20%20%3Fd%20a%20"\
-    "meshv%3ADescriptor%20.%0D%0A%20%20%3Fd%20rdfs%3Alabel%20%3FdName%0D%0A%20%20FILTER(REGEX(%3FdName%2C%27"
-  END_SPARQL_MESH_URI = "%27%2C%20%27i%27))%20%0D%0A%7D%20%0D%0AORDER%20BY%20%3Fd%20%0D%0A"
   LCSH_BASE_URI = "http://id.loc.gov/authorities/subjects/suggest/?q="
   LCSH_ID_URI = "http://id.loc.gov/authorities/subjects/"
   MESH_ID_URI = "https://id.nlm.nih.gov/mesh/"
+
   MEMOIZED_MESH_FILE = "memoized_mesh.txt"
   MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
   MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
-  SEARCHABLE_MESH_FILE = ENV['SUBJECTS_MESH_FILE']
-  SEARCHABLE_LCSH_FILE = ENV['SUBJECTS_LCSH_FILE']
+
   SEARCHABLE_LCNAF_FILE = "subjects_lcnaf.csv"
+  SEARCHABLE_MESH_FILE = 'subjects_mesh.jsonl'
+  SEARCHABLE_LCSH_FILE = 'subjects_lcsh.jsonl'
 
   def initialize
     puts "initializing header_lookup..."
     # these are the terms to search through for header lookups
-    @@searchable_mesh_terms ||= YAML.load_file(SEARCHABLE_MESH_FILE)
-    @@searchable_lcsh_terms ||= YAML.load_file(SEARCHABLE_LCSH_FILE)
+    @@searchable_mesh_file ||= SEARCHABLE_MESH_FILE
+    @@searchable_lcsh_terms ||= SEARCHABLE_LCSH_FILE
```

Comment:
minor: Typo on consistent name (both end in `_file` or both end in `_terms`)

Reply:
It's kind of concerning that this was still passing specs, but I guess that just means it was a consistent misspelling heh.